lib/board.rb

```diff
@@ -0,0 +1,75 @@
+class Board
+  attr_reader :board
+
+  def initialize
+    # @board = [' ',' ',' ',' ',' ',' ',' ',' ',' ']
+    # @board = %W(\s \s \s \s \s \s \s \s \s)
+    play_again
+  end
+
+  def play_again
+    @board = Array.new(9, '-')
+  end
+
+  def display
+    puts
+    puts ' ' + board[0] + ' │ ' + board[1] + ' │ ' + board[2] + '    ░   1   2   3'
+    puts '———┼———┼———' + (' ' * 3) + '░'
+    puts ' ' + board[3] + ' │ ' + board[4] + ' │ ' + board[5] + '    ░   4   5   6'
+    puts '———┼———┼———' + (' ' * 3) + '░'
+    puts ' ' + board[6] + ' │ ' + board[7] + ' │ ' + board[8] + '    ░   7   8   9'
+    puts
```

Comment:
As indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png), **no puts/gets** in **lib** folder

Reply:
done, thanks